pkg/workflows/execution/interpolation.go

```diff
@@ -0,0 +1,100 @@
+package execution
```

Comment:
Maybe consider `exec`?
```suggestion
package exec
```
https://go.dev/blog/package-names